### PR TITLE
Clarify purpose of ci steps

### PR DESCRIPTION
**Pull request by dlawin**
These were creating some questions when unexplained

### CODE REVIEW

File: docs/guides/ci_guides/dbt_core/gitlab_ci.md [hunk 1]
```diff
@@ -67,10 +71,22 @@ run_pipeline:
...
+
+    # Install dbt packages defined in packages.yml
     - dbt deps
+
+    # Run and test modified dbt models and models downstream of them
+    # Otherwise defer to production for unmodified models
     - dbt build --select state:modified+ --defer --state ./ --exclude config.materialized:snapshot --profiles-dir ./
+
+    # Use the Datafold sdk to create a diff and write results to the PR
     - datafold dbt upload --ci-config-id 26 --run-type $TYPE --commit-sha $CI_COMMIT_SHA
```
Comment (kylemcnair) [inline docs/guides/ci_guides/dbt_core/gitlab_ci.md:87]: Is it worth calling out the significance of the config ID or how to find it for your respective config?

Screen Shot 2022-11-08 at 9 21 59 AM

**Review by kylemcnair — APPROVED** [review]
these comments are helpful 👌

**Comment by williebsweet** [comment]
@dlawin We might need to reference the CI Guides at the bottom of the [dbt Core](https://docs.datafold.com/integrations/orchestration/dbt_core) setup guide.

Something like, "To complete the dbt Core integration, add Datafold to your CI workflow as outlined in our CI Guides (link to guide)."

I think the same customer conversation that prompted this PR also made me realize that the current dbt Core guide does not actually end with a functioning setup because we need them to also add datafold to their jobs.

**Comment by dlawin** [comment]
> @dlawin We might need to reference the CI Guides at the bottom of the [dbt Core](https://docs.datafold.com/integrations/orchestration/dbt_core) setup guide.
> 
> Something like, "To complete the dbt Core integration, add Datafold to your CI workflow as outlined in our CI Guides (link to guide)."
> 
> I think the same customer conversation that prompted this PR also made me realize that the current dbt Core guide does not actually end with a functioning setup because we need them to also add datafold to their jobs.

It probably makes sense to link back in the other direction as a prerequisites section too. Realizing we're assuming they've created an integration when coming into the doc

File: docs/guides/ci_guides/dbt_core/gitlab_ci.md (hunk 1 above)
Comment (dlawin) [reply docs/guides/ci_guides/dbt_core/gitlab_ci.md:93]: I've added a prereq that links to where to find this

**Pull request merged**